main.go

```diff
@@ -31,28 +31,37 @@ func init() {
 func main() {
 	flag.Parse() // required for glog flags and testing package flags
 
-	v := viper.New()
-	config.SetupViper(v, "pbs")
-	cfg, err := config.New(v)
+	cfg, err := loadConfig()
 	if err != nil {
 		glog.Fatalf("Configuration could not be loaded or did not pass validation: %v", err)
 	}
-	if err := serve(Rev, cfg); err != nil {
+
+	err = serve(Rev, cfg)
+	if err != nil {
 		glog.Errorf("prebid-server failed: %v", err)
 	}
 }
 
+func loadConfig() (*config.Configuration, error) {
```

Comment:
Moved the Viper setup logic into its own method to mirror the encapsulation of the server setup. IMHO, this seems easier to read.